app/src/main/kotlin/com/simplemobiletools/applauncher/activities/MainActivity.kt

```diff
@@ -33,22 +34,20 @@ class MainActivity : SimpleActivity(), RefreshRecyclerViewListener {
     private var mStoredPrimaryColor = 0
     private var mStoredTextColor = 0
 
+    private var emptyViews: ArrayList<View>? = null
```

Comment:
doesnt have to be null, lets just make it empty

Reply:
actually we dont even need it, lets just do it like here https://github.com/SimpleMobileTools/Simple-App-Launcher/blob/master/app/src/main/kotlin/com/simplemobiletools/applauncher/activities/SettingsActivity.kt#L27

Reply:
or just create it locally at maybeShowEmptyView to avoid duplication

Reply:
done :+1: